src/webhook/validation/preview.go

```diff
@@ -18,6 +18,9 @@ func oneAgentModePreviewWarning(dv *dynakubeValidator, dynakube *dynatracev1beta
 	} else if dynakube.ApplicationMonitoringMode() && dynakube.NeedsCSIDriver() {
 		log.Info("DynaKube with applicationMonitoring was applied, warning was provided.")
 		return fmt.Sprintf(featurePreviewWarningMessage, "applicationMonitoring")
+	} else if dynakube.HostMonitoringMode() && dynakube.NeedsCSIDriver() {
```

Comment:
I don't think this feature is in preview, a warning that the CSI driver is missing is enough.